libs/flowbite-angular/dropdown/dropdown.theme.ts

```diff
@@ -57,7 +57,7 @@ export interface DropdownTheme {
  */
 export const dropdownTheme: DropdownTheme = createTheme({
   root: {
-    base: '',
+    base: 'relative inline w-fit',
```

Comment:
look for a way to remove w-fit and make wdth the same size as the button if possible